### PR TITLE
fix(app): suppress title bar exit in calcheck results

### PR DESCRIPTION
**Pull request by sfoster1**
And also remove the unused parts (like 90%) of styles.css in the
calcheck hierarchy.

Note: I haven't tested this yet

### CODE REVIEW

File: app/src/components/CheckCalibration/index.js [hunk 1]
```diff
@@ -160,7 +162,14 @@ export function CheckHealthCalibration(
 
   const titleBarProps = {
     title: ROBOT_CALIBRATION_CHECK_SUBTITLE,
-    back: { onClick: confirmExit, title: EXIT, children: EXIT },
+    back: {
+      onClick: confirmExit,
+      title: EXIT,
+      children: EXIT,
```
Comment (b-cooper) [inline app/src/components/CheckCalibration/index.js:168]: if you wanted to completely get rid of the `styles.css` file you could replace the className with a dummy box e.g. 
```
back: currentStep === Sessions.CHECK_STEP_RESULTS_SUMMARY ? {title: null, children: (<Box width={SIZE_1} />)} : {...}
```

Comment (sfoster1) [reply app/src/components/CheckCalibration/index.js:168]: I think I tried that in some other case and it had some display issues around active-state rendering. I think we should just keep the styles thing until that's a styled-component or primitive and we can use a css literal or something

**Review by b-cooper — APPROVED** [review]
🔑

**Review by Laura-Danielle — APPROVED** [review]
works for me.

**Pull request merged**